src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -43,16 +43,12 @@ class Filter extends Component<Props> {
       hasForSaleArtworks: artist.counts.for_sale_artworks > 0,
       hasBuyNowArtworks: artist.counts.ecommerce_artworks > 0,
       hasAuctionArtworks: artist.counts.auction_artworks > 0,
+      hasArtworks: artist.counts.artworks > 0,
     }
   }
 
   get showZeroState() {
-    const showZeroState =
-      !this.existy.hasAuctionArtworks &&
-      !this.existy.hasBuyNowArtworks &&
-      !this.existy.hasForSaleArtworks
-
-    return showZeroState
+    return !this.existy.hasArtworks
```

Comment:
I could have sworn making this update to the zero state at some point...this is probably a better check.

Reply:
I remember this change!